.github/workflows/nightly.yaml

```diff
@@ -28,20 +28,14 @@ jobs:
         with:
           java-version: 11
 
-      - name: Cache gradle dependencies
+      - name: Test
         uses: burrunan/gradle-cache-action@v1
```

Comment:
Looks like gradle-caching-runner or the like will be a good name. Not surprising to see the best caching comes with having full visibility to Gradle.

Reply:
Naming is hard :)
Thanks for the suggestion. 

I see JUnit team uses the following, and it [can't be folded to a single command](https://github.com/junit-team/junit5/blob/4688072d1722f4906152351083e0133ae4e78feb/.github/workflows/main.yml#L44-L46), so I'm exploring ways to allow multiple (Gradle? Shell?) commands to be executed (to avoid copy-paste the boring `uses:...`)

```yaml
run: |
  ./gradlew --version
  ./gradlew build
```